app/controllers/favorites_controller.rb

```diff
@@ -1,32 +1,33 @@
 class FavoritesController < ApplicationController
   include ActionView::Helpers::TextHelper
-  
+
   def index
     pet_ids = favorite.contents.keys
-    @pets = Pet.find(pet_ids)
+    @favorite_pets = Pet.find(pet_ids)
```

Comment:
I feel like this is something we can look into refactoring later

Reply:
Wasn't one of these supposed to be a Pet.all?

Reply:
is what you are looking for on line 7?